container-storage-setup.spec

```diff
@@ -15,10 +15,7 @@ URL:            http://github.com/projectatomic/container-storage-setup/
 Source0: %{git0}/archive/%{commit}/%{repo}-%{shortcommit}.tar.gz
 BuildArch: noarch
 
-BuildRequires:  pkgconfig(systemd)
```

Comment:
We still have systemctl commands. So we atleast have install time/run time depedency on systemd. (If not build time dependency).